aten/src/ATen/native/sparse/cuda/SparseSemiStructured.cpp

```diff
@@ -0,0 +1,11 @@
+#include <torch/library.h>
+#include <torch/types.h>
+
+TORCH_LIBRARY_FRAGMENT(sparse, m) {
```

Comment:
I don't think this is something we typically land in aten. We'd probalby want to make these private natiev functions.